### ISSUE TITLE
ks param set is broken

### ISSUE THREAD

**Issue by ankushagarwal** [ksonnet/ksonnet#381]
/kind bug

**What happened**:
Ran `ks param set workflows namespace kubeflow-releasing --env=releasing`
Got an error message:
`ERROR Invalid params schema -- did not expect node type: *ast.ApplyBrace`

**What you expected to happen**:
No error message

**How to reproduce it (as minimally and precisely as possible)**:
```
git clone https://github.com/kubeflow/kubeflow.git
cd kubeflow/components/tensorflow-notebook-image/releaser
ks param set workflows namespace kubeflow-releasing --env=releasing
```

**Environment**:
- ksonnet version (use `ks version`):
ksonnet version: 0.9.1
jsonnet version: v0.9.5
client-go version: 1.8

**Comment by bryanl** [comment]
After taking a look at this, it isn't broken, the kubeflow environment is using a construct that ksonnet is unfamiliar with. I believe this won't be hard to accommodate, and it this new functionality will be included in 0.9.2.

**Comment by ankushagarwal** [comment]
Thank you!
